src/corehost/cli/ijwhost/corhdr.h

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+
+#ifndef CORHDR_H
```

Comment:
Can I assume you trimmed as much as possible?

Reply:
Yep. I only pulled in what I needed. The only extra pieces are extra members of some of the enums since I didn't think we would want to have partial enum definitions that didn't match coreclr in here.